src/test/acceptance/features/address-lookup.feature

```diff
@@ -61,3 +61,16 @@ Feature: Select address
     When I choose to change my address
     And I click the address not listed link
     Then I am shown the manual address page
+
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario Outline: Enter an invalid postcode on the postcode lookup page
+    Given I have entered my details up to the postcode page
+    When I enter in <postcode> as my postcode
```

Comment:
Not a blocker but probably don't need `in` here:
`I enter in <postcode> as my postcode` -> `I enter <postcode> as my postcode`

Reply:
Updated to remove `in`.